### PR TITLE
HuyenNguyen - Swift Basic - Bai 1,2,3,4,5,6,7

### PR DESCRIPTION
**Pull request by at-huyennguyen2**
2.1 Sphere Area
2.2 Spherical Volume
2.3 Quadratic Equation 2
2.4 - The sum of the first Fibonacy numbers
      - List the first 100 Fibonacy numbers
      - Calculate sin (x) using the Taylor Series
      - List the happy numbers
2.5  Find substring
2.6 Transposition random elements in the array
2.7 Given an array of integers, replace all the occurrences of elemToReplace with substitutionElem

### CODE REVIEW

File: 02_SwiftBasic/HuyenNguyen/Baitap2.2.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+func quadraticEquation( a: Float, b: Float, c: Float) -> Float {
+    let delta : Float = b * b - (4 * a * c)
+    if delta == 0 { print("x =", -b / (2 * a)) }
+    else if delta < 0 { print("no solution") }
+    else { print("x1 =", -b + sqrtf(delta) / (2 * a), "x2 = ", -b - sqrtf(delta) / (2 * a)) }
+    return delta
```
Comment (at-hainguyen3) [inline 02_SwiftBasic/HuyenNguyen/Baitap2.2.playground/Contents.swift:8]: Tại sao return về delta?

File: 02_SwiftBasic/HuyenNguyen/Baitap2.3.playground/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    let d: Int = a1 * b2 - a2 * b1
+    let dx: Int = c1 * b2 - c2 * b1
+    let dy: Int = a1 * c2 - a2 * c1
+    if d == 0 && dx + dy == 0 {
+        print("He phuong trinh co vo so nghiem")
+    } else if d == 0 { print("He phuong trinh vo nghiem") }
+    else {
+        print("He phuong trinh co nghiem x, y = ", dx / d,",", dy / d)
+    }
+    return 0;
```
Comment (at-hainguyen3) [inline 02_SwiftBasic/HuyenNguyen/Baitap2.3.playground/Contents.swift:13]: Tại sao return về 0

File: 02_SwiftBasic/HuyenNguyen/Baitap2.6.playground/Contents.swift [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import UIKit
+
+var arr: [Int] = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+arr.shuffle()
```
Comment (at-tienle) [inline 02_SwiftBasic/HuyenNguyen/Baitap2.6.playground/Contents.swift:4]: Khổ tâm nhĩ

**Comment by at-tienle** [comment]
Do lâu rồi ko thấy fix comment chi

**Pull request closed without merging**